couchbase/src/main/scala/akka/stream/alpakka/couchbase/scaladsl/CouchbaseFlow.scala

```diff
@@ -18,12 +17,13 @@ object CouchbaseFlow {
    * Create a flow to query Couchbase for by `id` and emit [[com.couchbase.client.java.document.JsonDocument JsonDocument]]s.
    */
   def fromId(sessionSettings: CouchbaseSessionSettings, bucketName: String): Flow[String, JsonDocument, NotUsed] =
-    Setup
-      .flow { materializer => _ =>
-        val session = CouchbaseSessionRegistry(materializer.system).sessionFor(sessionSettings, bucketName)
-        Flow[String]
-          .mapAsync(1)(id => session.flatMap(_.get(id /* timeout? */ ))(materializer.system.dispatcher))
-          .collect { case Some(doc) => doc }
+    Flow
+      .setup {
+        case (materializer, _) =>
```

Comment:
Here and below, it can be a Function instead of a PartialFunction - drop the `case`.